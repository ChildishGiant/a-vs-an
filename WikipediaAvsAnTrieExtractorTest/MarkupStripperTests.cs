using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ExpressionToCodeLib;
using WikipediaAvsAnTrieExtractor;
using Xunit;

namespace WikipediaAvsAnTrieExtractorTest {
    
    public class MarkupStripperTests {
        readonly RegexTextUtils utils = new RegexTextUtils();

        [Fact]
        public void StripsMarkup()
        {
            var data = new Dictionary<string, string>
            {
            };
            foreach (var kvp in data) {
                PAssert.That(() => kvp.Value == utils.StripWikiMarkup(kvp.Key));
            }
        }
    }
}
