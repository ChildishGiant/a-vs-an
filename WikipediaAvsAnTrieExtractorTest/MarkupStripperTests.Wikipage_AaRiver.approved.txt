Aa River may refer to:

Aa (France), in northern France
Aa (Meuse), in North Brabant, Netherlands
Aa of Weerijs, in North Brabant, Netherlands
Drentsche Aa, a river in Groningen, Netherlands
Aabach (Greifensee), or Ustermer Aa, a river in Switzerland
Aabach (Afte), formerly called the Große Aa, a river in Germany
Aa, a river in Antwerp, Belgium which joins the Nete River

 Former names ==

Gauja, a river in Latvia, formerly known as Livländische Aa
Lielupe, a river in Latvia, formerly known as Kurländische Aa





